src/components/header/help-modal/index.css

```diff
@@ -15,7 +15,7 @@
 }
 
 .help-modal .doc-link {
-  display: block;
+  display: inline-block;
```

Comment:
Is this necessary with the new div?

Reply:
yes it is. display: block is again gonna make the anchor tag utilise the whole of the width again

Reply:
Can you just delete the line?

Reply:
Links are by default inline. 

The other option is to not use the div and just set the display of the link.

Reply:
Make the anchor tags inline-block , anchor tags are inline and won't accept any styles , we only need to change the style from block to inline-block , no need for adding <div> tags

Reply:
This is an interesting one actually, if we set to inline-block we will get 2 links side by side , we need to set them to a inline-block and break lines after each ```<a>``` , so the solution can be to use ```<br/>``` or using inline-table or use this https://css-tricks.com/injecting-line-break/